### ISSUE TITLE
license mismatch

### ISSUE THREAD

**Issue by brlcad** [kovacsv/VisualScriptCAD#2]
Unless they've been relicensed, the CSG portions from CGAL that you're using are GPL.  That would make this a derivative work subject to the terms of the GPL too. :/

**Comment by brlcad** [comment]
For what it's worth, I'd much rather see this code be MIT-licensed.  There are currently limited options for robust evaluations (even CGAL's fails sometimes), but maybe something to keep an eye out on.  I was excited to find your project, but can't do anything with GPL code.

**Comment by kovacsv** [comment]
Yeah, I know :( By the way [the node engine behind it](https://github.com/kovacsv/VisualScriptEngine) is still MIT.
